database/pgDatabase.js

```diff
@@ -13,14 +18,19 @@ Price.init({
     unique: true,
     primaryKey: true,
   },
-  basePrice: DataTypes.DECIMAL,
+  basePrice: DataTypes.INTEGER,
   discountPercentage: DataTypes.INTEGER,
-  saleEndDate: DataTypes.DATE,
+  saleNumOfDays: DataTypes.INTEGER,
```

Comment:
do you plan to update this daily as the end date grows closer? If you would prefer to store an integer, consider storing the millisecond representation of the date instead of the number of days from the time you create/update the record to the end date, since that requires a lot of upkeep

Reply:
this is a good point. When Jason was doing his DB he just added three days from day of data generation so i didnt think about that. I think I'll probably take your suggestion and save the end date as an integer in milliseconds.